kubernetes/apps_kubeclient_blackbox_test.go

```diff
@@ -199,6 +199,24 @@ func (testMetrics) GetMemoryMetricsRange(pods []v1.Pod, namespace string, startT
 	return nil, nil // TODO
 }
 
+func (testMetrics) GetNetworkSentMetrics(pods []v1.Pod, namespace string, startTime time.Time) (*app.TimedNumberTuple, error) {
+	return nil, nil // TODO add fake impl when tests exercise this code
```

Comment:
shouldn't you return something else than `nil, nil` here ?

Reply:
This is just to implement all of the methods in the fake MetricsInterface for the KubeClient tests. There aren't yet any tests that exercise these methods, so this is just a placeholder to allow the test to compile. I plan to add additional tests shortly.

Reply:
ok